### PR TITLE
Rework the full doc for the pane  block

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #1083 

- Redo doc for pane
- Update Release Notes
- "Better"? buttons for Reference|VisElements
- other tiny fixes

### CODE REVIEW

File: docs/refmans/gui/viselements/generic/pane-as-page-l.png
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/generic/pane-as-page-l.png:1]: why is it not 5.0 like the dark one ?

**Review by FredLL-Avaiga — APPROVED** [review]
🥇

**Review by jrobinAV — APPROVED** [review]
I am sure this T-shaped snake logo is about to be trendy again 30 years from now!

File: docs/refmans/gui/viselements/generic/pane-as-page-l.png
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/generic/pane-as-page-l.png:1]: Yeah I saw that at the last minute.... I will redo it. Thx

**Pull request merged**